conan/cli/commands/create.py

```diff
@@ -102,7 +96,8 @@ def create(conan_api, parser, *args):
                                                       clean=args.lockfile_clean)
 
     conan_api.lockfile.save_lockfile(lockfile, args.lockfile_out, cwd)
-    return deps_graph
+    return {"graph": deps_graph,
+            "conan_api": conan_api}
```

Comment:
`conan_api` is not used for now. I kept it just for future implementations.

Reply:
Just like `graph info` does right?

Reply:
``conan_api`` not used, it can be removed.
Actually, it is only the serialization of ``graph`` that changes, so this doesn't need to be changed.